src/scripts/components/sendForm.component.jsx

```diff
@@ -90,6 +90,7 @@ class SendForm extends Component {
 						</div>
 						<textarea className="pure-input-1" type="text" placeholder="Page description" name="description" maxLength="150" rows="5" required="true"></textarea>
 						<textarea className="pure-input-1" type="text" placeholder="Custom message (optional)" name="message" maxLength="250" rows="3"></textarea>
+						<textarea className="pure-input-1" type="text" placeholder="Custom message page name (optional)" name="pagename" maxLength="50"></textarea>
```

Comment:
Hey @SekoiaTree! Sorry for taking so long to get around to this.  I've got the changes to support this on the serverside ready to go! I'd like the description to be a bit different here and to use an input instead of a textarea since it's only 50 chars.
```suggestion
						<input className="pure-input-1" type="text" placeholder="File Name (optional) (ie. site.com/url-path.html) (defaults to page title)" name="pagename" maxLength="50"></input>
```
I'll go ahead and commit that change on this branch and merge this change. 
Thanks for the suggestion again!